kuma/install.go

```diff
@@ -47,6 +64,28 @@ func (kuma *Kuma) installKuma(del bool, namespace string, version string) (strin
 	}
 	return status.Installed, nil
 }
+func (kuma *Kuma) applyHelmChart(del bool, namespace string) error {
+	kClient := kuma.MesheryKubeclient
+	if kClient == nil {
+		return ErrNilClient
+	}
+	kuma.Log.Info("Installing using helm charts...")
+	version := getLatestKumaChartVersion()
```

Comment:
@Revolyssup do we need to get the release version here as well?
Function `installKuma` does have a `version` argument which is in fact latest release of kuma obtained when the adapter starts.

Reply:
See: https://github.com/meshery/meshery-kuma/blob/7c060fc0605046e209c7576253102ef0a56fa118/internal/config/operations.go#L14

Reply:
Oh I see. They store their helm charts in a different github repo :thinking:

Reply:
@Revolyssup Let's try to keep a single source for obtaining kuma's version. In this approach, `applyHelmChart` would ignore version requested while calling `installKuma` and install the latest release of kuma. This is not desired.

Reply:
@Revolyssup why not just use `AppVersion` of `ApplyHelmChart`?

Reply:
@DelusionalOptimist  not fetching the version on the fly now and using user passed version to config.
Before you ask, why am I converting user passed app version into chart version instead of directly passing the appVersion.
The ApplyHelmChart function internally does some normalization and adds "v" in front of version, which is the case in many service meshes but kuma doesn't follow this. So passing apiVersion directly would result in normalization and hence result in error. Utkarsh added a new function in latest release of meshkit which allows me to pass appVersion and get chartVersion from there.